sources/packages/web/src/views/aest/institution/ApproveDenyDesignation.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <!-- CSS class temporary-modal is work around for lack of responsiveness of v-dialog. -->
+  <ModalDialogBase :showDialog="showDialog" @dialogClosed="dialogClosed">
+    <template v-slot:content>
+      <v-container class="temporary-modal">
+        <formio
+          formName="approvedenydesignation"
+          :data="designation"
+          @loaded="formLoaded"
+          @submitted="submitDesignationUpdate"
```

Comment:
Why do we need the _@submitted="submitDesignationUpdate"_ and the _@click="submitDesignation()"_.
Who is submitting the form?